.github/workflows/autobump.yml

```
@@ -0,0 +1,55 @@
+name: Bump formulae on schedule or request
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/autobump.yml
+  workflow_dispatch:
+    inputs:
+      formulae:
+        description: Custom list of portable-ruby formulae to livecheck and bump if outdated
+        required: false
+  schedule:
+    # Daily at midnight
+    - cron: "0 0 * * *"
+
+env:
+  HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+
+permissions:
+  contents: read
+
+jobs:
+  autobump:
+    if: github.repository == 'Homebrew/homebrew-portable-ruby'
```

```suggestion
    if: github.repository_owner == 'Homebrew'
```

Minor but more concise

---

yeah, works for me 👍